props/saber_BC_buttons.h

```diff
@@ -425,6 +1423,26 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
   SaberBCButtons() : PropBase() {}
   const char* name() override { return "SaberBCButtons"; }
 
+#ifndef MENU_SPEC_TEMPLATE
+  virtual void EnterBladeLengthMode() {
+    if (scroll_presets_) return;
+#ifdef DYNAMIC_BLADE_LENGTH
+    if (!current_style()) return;
+    if (current_mode == this) {
+      sound_library_.SayEditBladeLength();
+        pushMode<MKSPEC<BCMenuSpec>::BCSelectBladeMenu>();
+    }
+#endif  // DYNAMIC_BLADE_LENGTH
+  }
+
+  void EnterVolumeMenu() {
```

Comment:
EnterVolumeMenu can be implemented (and be useful) even if MENU_SPEC_TEMPLATE is defined.

Reply:
I suppose so, as an additional / alternate / shortcut version sort of thing?
Unguarded.

Reply:
I wonder, if MENU_SPEC_TEMPLATE is defined, wouldn't it make more sense to use that volume menu:

   pushMode<MKSPEC<MENU_SPEC_TEMPLATE>::ChangeVolumeMode>()

?

Reply:
Oh I see, shortcut to the OS menu version. 
Yeah, ok. I'll add an announce too so it's known what happened.
Done.